{
    "data_version": "4.0",
    "data_type": "CVE",
    "data_format": "MITRE",
    "CVE_data_meta": {
        "ID": "CVE-2025-32908",
        "ASSIGNER": "secalert@redhat.com",
        "STATE": "PUBLIC"
    },
    "description": {
        "description_data": [
            {
                "lang": "eng",
                "value": "A flaw was found in libsoup. The HTTP/2 server in libsoup may not fully validate the values of pseudo-headers :scheme, :authority, and :path, which may allow a user to cause a denial of service (DoS)."
            }
        ]
    },
    "problemtype": {
        "problemtype_data": [
            {
                "description": [
                    {
                        "lang": "eng",
                        "value": "Misinterpretation of Input",
                        "cweId": "CWE-115"
                    }
                ]
            }
        ]
    },
    "affects": {
        "vendor": {
            "vendor_data": [
                {
                    "vendor_name": "Red Hat",
                    "product": {
                        "product_data": [
                            {
                                "product_name": "Red Hat Enterprise Linux 6",
                                "version": {
                                    "version_data": [
                                        {
                                            "version_value": "not down converted",
                                            "x_cve_json_5_version_data": {
                                                "defaultStatus": "unknown"
                                            }
                                        }
                                    ]
                                }
                            },
                            {
                                "product_name": "Red Hat Enterprise Linux 7",
                                "version": {
                                    "version_data": [
                                        {
                                            "version_value": "not down converted",
                                            "x_cve_json_5_version_data": {
                                                "defaultStatus": "unknown"
                                            }
                                        }
                                    ]
                                }
                            },
                            {
                                "product_name": "Red Hat Enterprise Linux 8",
                                "version": {
                                    "version_data": [
                                        {
                                            "version_value": "not down converted",
                                            "x_cve_json_5_version_data": {
                                                "defaultStatus": "affected"
                                            }
                                        }
                                    ]
                                }
                            },
                            {
                                "product_name": "Red Hat Enterprise Linux 9",
                                "version": {
                                    "version_data": [
                                        {
                                            "version_value": "not down converted",
                                            "x_cve_json_5_version_data": {
                                                "defaultStatus": "affected"
                                            }
                                        }
                                    ]
                                }
                            }
                        ]
                    }
                }
            ]
        }
    },
    "references": {
        "reference_data": [
            {
                "url": "https://access.redhat.com/security/cve/CVE-2025-32908",
                "refsource": "MISC",
                "name": "https://access.redhat.com/security/cve/CVE-2025-32908"
            },
            {
                "url": "https://bugzilla.redhat.com/show_bug.cgi?id=2359343",
                "refsource": "MISC",
                "name": "https://bugzilla.redhat.com/show_bug.cgi?id=2359343"
            }
        ]
    },
    "work_around": [
        {
            "lang": "en",
            "value": "Currently, no mitigation was found for this vulnerability."
        }
    ],
    "impact": {
        "cvss": [
            {
                "attackComplexity": "LOW",
                "attackVector": "NETWORK",
                "availabilityImpact": "HIGH",
                "baseScore": 7.5,
                "baseSeverity": "HIGH",
                "confidentialityImpact": "NONE",
                "integrityImpact": "NONE",
                "privilegesRequired": "NONE",
                "scope": "UNCHANGED",
                "userInteraction": "NONE",
                "vectorString": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H",
                "version": "3.1"
            }
        ]
    }
}